cmd/kraft/build/build.go

```diff
@@ -310,6 +310,9 @@ func (opts *Build) Run(cmd *cobra.Command, args []string) error {
 	if len(missingPacks) > 0 {
 		for _, p := range missingPacks {
 			p := p // loop closure
+			if newLen := len(unikraft.TypeNameVersion(p)) + len("pulling "); !norender && newLen > nameWidth {
+				nameWidth = newLen
+			}
```

Comment:
I'd add simply use the integer value to decrease processing time and a comment indicating that it refers to additional length of the string used by the left text name.

Also I believe "`configuring `" is the longest string used.

Reply:
Having said this, shouldn't this be added to [the pre-emptive calculation](https://github.com/unikraft/kraftkit/blob/staging/cmd/kraft/build/build.go#L134-L138)?

Reply:
The calculation there is wrong because it does not take into account the version.

Reply:
Not 100% sure why, but in the other cases it works (fetching), I can change there and try out again

It's used only